resources/nl-core/fhirpkg.lock.json

```diff
@@ -1,7 +1,8 @@
 {
```

Comment:
Better not to include this file. Also it contains an outdated depedency. Just let it be I would say.